src/components/base/Sitenav.vue

```diff
@@ -28,8 +28,8 @@ export default {
           icon: 'chevron-left'
         },
         manage: {
-          title: 'Manage',
-          link: 'manage',
+          title: 'Contact Report',
+          link: 'contact-report',
           icon: 'border-all'
```

Comment:
Please change the icon to a more suitable

Reply:
which one do you suggest?

Reply:
icon: "print"